test/cpp/jit/test_misc.cpp

```diff
@@ -520,6 +520,14 @@ TEST(SchemaParserTest, NestedArrays) {
                                               .getElementType()));
 }
 
+TEST(SchemaParserTest, OutVariant) {
```

Comment:
Could you add more test cases including:
1) operator without any out argument
2) `is_write` argument but not kwonly
3) arguments that are not `out` are correctly distinguished by `is_out`

Reply:
will do thanks